src/lib/components/AnimatedLogo/AnimatedLogo.module.css

```diff
@@ -0,0 +1,146 @@
+.logo {
+  transform: scale(1);
+  transition: transform var(--transition-duration)
+    var(--transition-timing-function);
+  animation: scale var(--spin-time) ease-in-out infinite;
+}
+
+.logo svg {
+  width: 94%;
+  height: auto;
+  margin: 3%;
+  transform: scale(1);
+  transition: transform var(--transition-duration)
+    var(--transition-timing-function);
+}
+
+.circles {
+  height: var(--circle-size);
+  width: var(--circle-size);
+  position: absolute;
+  margin: 0 auto;
+  top: 50%;
+  left: 50%;
+  transform: translate3D(-50%, -50%, 0) scale(1);
+  transition: transform var(--transition-duration)
+    var(--transition-timing-function);
+  cursor: pointer;
+}
+
+.circles.circles_shadowed .circle.third {
+  box-shadow: 0 0 var(--shadow-size) 0 rgba(#00b400, 0.7),
+    0 0 var(--shadow-size) 0 rgba(0, 255, 0, 0.5) inset;
+}
+
+.circles.circles_shadowed .circles:active .circle.third {
+  box-shadow: 0 0 var(--shadow-size) 0 rgba(0, 180, 0, 1),
+    0 0 var(--shadow-size) 0 rgba(0, 255, 0, 0.6) inset;
+}
+
+.circles.circles_shadowed .circles:active .circle.third {
+  box-shadow: 0 0 var(--shadow-size) 0 rgba(255, 255, 255, 0.5) inset;
+}
+
+.circle {
+  box-sizing: border-box;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  border: solid transparent;
+  border-width: var(--circle-stroke);
+  border-radius: 115% 140% 145% 110%/125% 140% 110% 125%;
+  mix-blend-mode: overlay;
+  transition: all var(--transition-duration) var(--transition-timing-function);
+}
+
+.circle.first {
```

Comment:
could not add postcss-nesting. It doesn't work for me)
If you have time to help — welcome:)